app/src/home.js

```diff
@@ -242,7 +244,79 @@ export default function Home() {
     });
   }, [ ])
 
+  const handleSave = (e) => {
+    e.preventDefault()
+    saveSession()
+  }
+
+  async function createSession(data) {
+    const apiName = 'MTADemoAPI';
+    const path = 'createSession';
+    const myInit = { 
+    //   headers: { 
+    //     Authorization: `Bearer ${(await Auth.currentSession()).getIdToken().getJwtToken()}`,
+    //   },
+      response: true,
+      queryStringParameters: data
+    };
+
+    const result =  await API.post(apiName, path, myInit); 
+
+    return result;
+  }
 
+  const saveSession = () => {
+    const id = uuid();
+    Storage.configure({
+      bucket: 'mtastack-mtastackstorages3bucketc161f3b3-1tfncqzctldb1',
+      level: 'public',
+      region: 'us-west-2',
+    });
+    const transcribeAddress = `${id}/transrcibe-medical-output/transcribe.txt`
+    const comprehendAddress = `${id}/comprehend-medical-output/comprehend.txt`
+
+    var transcripts_texts = "";
+    transcripts.forEach((item) => { transcripts_texts += item.text + " "});
+    Storage.put(transcribeAddress, transcripts_texts);
+
+    var comprehend_texts = "";
+    const allResults = [].concat(...comprehendResults)
+    const filteredResultsM =  allResults.filter(r => r.Category === 'MEDICATION');
+    comprehend_texts += 'Medications/n'
+    filteredResultsM.map((r,i) => comprehend_texts+=r.Text + 
+    '/n');
+    const filteredResultsMC =  allResults.filter(r => r.Category === 'MEDICAL_CONDITION');
+    comprehend_texts += 'Medical Conditions/n'
+    filteredResultsMC.map((r,i) => {
+      comprehend_texts+=r.Text;
+      if(r.Attributes)
+        Object.keys(r.Attributes).map((i2) => comprehend_texts+=" "+r.Attributes[i2].Type+" "+r.Attributes[i2].Text+";");
+      comprehend_texts+='/n'
+    });
+    const filteredResultsTTP =  allResults.filter(r => r.Category === 
```

Comment:
Just to verify, does this also store the ICD-10,RX-norm  dictionaries from the Comprehend results?

Reply:
Yes, adn I modify the code so that it supports the Athena dependency. https://github.com/aws-samples/medical-transcription-analysis/blob/eafae50489bbda5b2cd751d3679ef7a7ae8d11ce/app/src/home.js#L282-L297